dts/bindings/display/led-strip-matrix.yaml

```diff
@@ -0,0 +1,86 @@
+# Copyright (c) 2024 TOKITA Hiroshi
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  Generic LED strip matrix (LED strip arranged in a grid pattern)
+
+compatible: "led-strip-matrix"
+
+include: display-controller.yaml
+
+properties:
+  led-strip:
+    type: phandle
+    required: true
+    description: |
+      Specify the LED strip that is the substance of the matrix.
```

Comment:
shouldn't this support potentially multiple strips? One may split the load into several GPIOs for improved frame rate in case of very long strips.

Reply:
Since the support for parallel execution is complicated, I would like it to be an another issue, but I modified the interface to make it possible.